cmd/sidecars/kafka/client/writer.go

```diff
@@ -45,16 +45,16 @@ func (writer *Writer) WriteMessage(channel string, message interface{}) error {
 	logger.Info("trying to write message in topic",
 		zap.String("channel", channel),
 		zap.String("resolved channel", resolvedCh))
-	go deliveryReport(writer.producer)
 
 	if errProduceMessage := writer.produceMessage(message, kafkaTopic(resolvedCh)); errProduceMessage != nil {
 		logger.Error("error while producing message",
 			zap.Any("error", errProduceMessage))
 		return errProduceMessage
 	}
 
-	writer.producer.Flush(flushTimeout)
-
+	logger.Info("flusing the producer")
+	writer.producer.Flush(10)
```

Comment:
Removed the const `flushTimeout` usage but didn't remove it from the code, maybe instead of putting the value itself there you could leave the `flushTimeout` as it is and just change its value to 10 milliseconds.

Reply:
yeah, will do